tapalcatl_server/handler.go

```diff
@@ -468,7 +471,14 @@ func MetatileHandler(p Parser, metatileSize int, mimeMap map[string]string, stor
 		reqState.Coord = &parseResult.Coord
 		reqState.HttpData = &parseResult.HttpData
 
-		metaCoord, offset := parseResult.Coord.MetaAndOffset(metatileSize)
+		metaCoord, offset, err := parseResult.Coord.MetaAndOffset(metatileSize, tileSize)
+		if err != nil {
+			configErrors.Add(1)
+			logger.Warning(LogCategory_ConfigError, "MetaAndOffset could not be calculated: %s", err.Error())
+			http.Error(rw, err.Error(), http.StatusInternalServerError)
+			reqState.ResponseState = ResponseState_Error
+			return
```

Comment:
True, I assumed that's what it should be based on updating the expvar. Is that `FetchState_ConfigErr`fetch state used anywhere? If not, maybe better to just get rid of it and leave the fetch state unset to have it default to nil then.